waterlog/src/Components/Map.js

```diff
@@ -41,8 +132,21 @@ export default class MapComponent extends Component {
 	}
 
 	render() {
+    const { error, loading, mapData } = this.props;
+    if (error) {
+			return <div>Error! {error.message}</div>;
```

Comment:
indentation and remove console log

Reply:
I removed console log, and fixed indentation